perceval/backends/core/confluence.py

```diff
@@ -91,6 +92,16 @@ def fetch(self, category=CATEGORY_HISTORICAL_CONTENT, from_date=DEFAULT_DATETIME
 
         return items
 
+    @staticmethod
+    def add_ancestors(url_prefix, item, content):
```

Comment:
the logic of `add_ancestor` could be simplified. For instance, at first you could check if the ancestors don't exist and return an empty list. Then, the next statements could tackle the case when the ancestors do exist.